content/posts/2020-06-01-hello-world/index.md

```diff
@@ -1,50 +1,22 @@
 ---
-title: Hello World 2
-author: Joe R
-twitter: loadsmartus
+title: Hello, world!
+author: Engineering
+twitter: LoadsmartEng
 layout: post
 lang: en
-path: /blog/hello-world/
-date: '2020-06-01'
-tags: ['blogs']
+path: /blog/hello-world
+date: "2020-06-01"
 ---
 
-Hello World from the other side.
+In 2014, Loadsmart was born. At that time, we had lots of ideas in mind about what we wanted to achieve and about everything we were planning to build to solve the challenges many companies face in the logistics industry. We knew the impact could be big. In a field where so many processes are still to the present date made by repetitive tasks, phone calls, and an extensive list of touch points, technology is the only option to scale efficiently.
```

Comment:
At that time, we had lots of ideas about how we wanted to achieve everything we were planning to build to solve the many challenges logistics companies face.